### ISSUE TITLE
Exception thrown on Special:SpecialPages

### ISSUE THREAD

**Issue by Aspallar** [NilsEnevoldsen/ScryfallLinks#41]
MediaWiki  1.34.0
PHP   7.2.24-0ubuntu0.18.04.3 (apache2handler)
MySQL  5.7.29-0ubuntu0.18.04.1

The SpecialPages entry in extension.json is causing Special:SpecialPages to throw **Fatal exception of type "InvalidArgumentException"**

```json
       "SpecialPages": {
                "DownloadDeck": [
                        "MediaWiki\\Extension\\ScryfallLinks\\SpecialDownloadDeck"
                ]
        },
```

Everything appears to work fine when it is removed.

Regards
Aspallar

**Comment by NilsEnevoldsen** [comment]
I'm not sure why that error is happening, but at this point it's safe to remove that Special Page placeholder entirely. It only existed as a transitional feature to keep old decklists from breaking before the pages were refreshed. I'll do that.

**Comment by Aspallar** [comment]
Fast reply. Nice extension :)

Looking at the source I figured that it would be safe to remove, but my knowledge is .... somewhat limited.

Regards
Aspallar

**Comment by NilsEnevoldsen** [comment]
Fixed in b1add1c7573921f614f99a9aa271d007c96080e2.
